ansible/roles_ocp_workloads/ocp4_workload_serverless/tasks/remove_workload.yml

```diff
@@ -114,15 +114,13 @@
     name: "{{ _ocp4_workload_serverless_install_plan_name }}"
     namespace: openshift-operators
 
-- name: Remove remaining knativeservings CRDs
+- name: Remove CatalogSource for mirrored catalog
```

Comment:
Should there be a conditional on this task? We don't really need to remove this catalog source unless it was deployed?

Reply:
But this is a idempotent statement. If it wasn't there then `state: absent` will already be true.